### ISSUE TITLE
Equality with threshold

### ISSUE THREAD

**Issue by graemeleehickey** [bozenne/BuyseTest#7]
It occurs to me in the following lines of code that the pairwise comparison does not match what was done in Buyse (2010).

https://github.com/bozenne/BuyseTest/blob/087f54d575ab5d10d0f8e06b199173f139b5057a/src/FCT_calcOnePair.h#L48:L55

Namely, if $X - Y > \tau$, Table II in Buyse declares this as "favourable". However, in the code here, we have $X - Y \ge \tau$.

The difference is subtle, but I thought I would highlight in case you wanted to make this clear in the documentation(?).

If you do change any code to "correct", would it be possible to have an option to keep the old threshold, i.e. >=?

**Comment by bozenne** [comment]
Thanks, I have clarified that in the documentation:
-BuyseTest function: "A pair will be classified as neutral if the difference in endpoint is strictly below this threshold. "
-BuyseTest-package, summary function for BuyseTest have now some text describing the net benefit with formula (where the >= appears).
See new version on Github in the branch master.

The reason for using >= instead of > is that when using a threshold of 3 months, one usually expect that differences in survival of exactly 0 would count as a relevant difference while when using no threshold one usually expect that no difference does not count as relevant. This is why by default the threshold is set to a very small value (I think 10^{-12}).
This way the default setting does match table 2 of Buyse but it is true that when specifying a non-0 threshold there is a difference. We discussed that at some point with Marc Buyse and Julien Peron and we agree that was better like that.
So the software will stay like that.
